engine/runtime-language-arrow/src/test/java/org/enso/interpreter/arrow/AddArrowTest.java

```diff
@@ -0,0 +1,71 @@
+package org.enso.interpreter.arrow;
+
+import static org.junit.Assert.*;
+
+import org.graalvm.polyglot.Context;
+import org.graalvm.polyglot.io.IOAccess;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class AddArrowTest {
+  private static Context ctx;
+
+  @BeforeClass
+  public static void initEnsoContext() {
+    ctx =
```

Comment:
Use [ContextUtils](https://github.com/enso-org/enso/blob/56b289ae792acce79501b96c4ff493899db25bbf/lib/java/test-utils/src/main/java/org/enso/test/utils/ContextUtils.java) and declare dependency `runtime-language-arrow/Test` --> `test-utils`. Being able to use context and project utils from `test-utils` was the main motivation to move `test-utils` into separate project in https://github.com/enso-org/enso/pull/10112